ccvm_simulators/solvers/dl_solver.py

```diff
@@ -149,6 +149,31 @@ def _calculate_grads_boxqp(self, c, s, q_matrix, v_vector, pump, rate, S=1):
         s_grads = -s_grad_1 + s_grad_2 - s_grad_3
         return c_grads, s_grads
 
+    def _calculate_grads_boxqp_adam(self, c, s, S=1):
```

Comment:
Let's hold this until an upcoming PR, too :)
(Same with this function in mf_solver)

Reply:
Sorry for the mess. They're just removed.